### PR TITLE
Ft api retrieve one post

### PR DESCRIPTION
**Pull request by ishimwebernard**
#### What does this PR do?
[what does this PR do?]
>this PR allows users to get one post from the database
#### Description of Task to be completed
[the tasks that you completed on the feature]
>added JavaScript method to trace the routes to get one posts
image

### CODE REVIEW

File: Server/config/mongo.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    mongoose.connect(process.env.MONGO_HASH,
+
+        { useUnifiedTopology: true,useNewUrlParser: true }).then(()=>{
+        
+        console.log(`Succesfully connected`);
+        
+        }).catch(error=>{
+        
+        console.log(String(error.message));
+
```
Comment (habinezadalvan) [inline Server/config/mongo.js]: return something like server error

File: Server/config/mongo.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+require("dotenv/config");
+
+const connectoToMongo = function(){
+    
+    mongoose.connect(process.env.MONGO_HASH,
+
+        { useUnifiedTopology: true,useNewUrlParser: true }).then(()=>{
+        
+        console.log(`Succesfully connected`);
```
Comment (habinezadalvan) [inline Server/config/mongo.js]: remove unnecessary console.log

File: Server/controllers/postsController.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+        static retrieveAllPosts =  function(request, response){
+            console.log(`Retrieving Comments ...`);
+            Posts.find().then(data=>{
+                response.json(data);
+                console.log("Data Succesfully retrieved");
+            }).catch(error=>{
+                console.log(error);
+                response.send("Something went wrong");
```
Comment (habinezadalvan) [inline Server/controllers/postsController.js]: return the response as a json object

File: Server/controllers/postsController.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            })
+        }
+        static getOne = function(request,response){
+            Posts.findById(request.params.postId).then(data=>{
+                response.json(data);
+            }).catch(error=>{
+                console.log(error);
+                response.send(`Something went wrong`);
```
Comment (habinezadalvan) [inline Server/controllers/postsController.js]: return the response as a json object

File: Server/controllers/postsController.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                response.json(data);
+                console.log("Data Succesfully retrieved");
+            }).catch(error=>{
+                console.log(error);
+                response.send("Something went wrong");
+            })
+        }
+        static getOne = function(request,response){
+            Posts.findById(request.params.postId).then(data=>{
+                response.json(data);
```
Comment (habinezadalvan) [inline Server/controllers/postsController.js:27]: add the status in your return

**Review by habinezadalvan — CHANGES_REQUESTED** [review]
Work on the comments below

File: Server/config/mongo.js (hunk 1 above)
Comment (ishimwebernard) [reply Server/config/mongo.js]: Unnecessary console.logs removed and returned something to the server

**Pull request merged**